<?xml version="1.0"?>

<launch>

	<rosparam file="$(find drones)/launch/params.yaml" command="load"/>

	<group ns="uav1">
	    <node name="drone_operator" pkg="drones" type="drone_operator" output="screen">
	        <remap from="/image" to="camera_uav/image_raw"/>
	        <remap from="/camera_info" to="camera_uav/camera_info"/>
	        <param name="uav_name" type="str" value="iris_1" />
	    </node>
	</group>

	<group ns="uav2">
	    <node name="drone_operator" pkg="drones" type="drone_operator" output="screen">
	        <remap from="/image" to="camera_uav/image_raw"/>
	        <remap from="/camera_info" to="camera_uav/camera_info"/>
	        <param name="uav_name" type="str" value="iris_2" />
	    </node>
	</group>

	<group ns="uav3">
	    <node name="drone_operator" pkg="drones" type="drone_operator" output="screen">
	        <remap from="/image" to="camera_uav/image_raw"/>
	        <remap from="/camera_info" to="camera_uav/camera_info"/>
	        <param name="uav_name" type="str" value="iris_3" />
	    </node>
	</group>

	<include file="$(find drones)/launch/animation.launch"/>

	<!--node pkg="plotjuggler" type="PlotJuggler" name="plotjuggler" output="screen" /-->

</launch>